docs/content/preview/drivers-orms/smart-drivers.md

```diff
@@ -110,11 +136,27 @@ For example, using the Go driver, you would set the parameters as follows:
     topology_keys=cloud1.region1.zone1,cloud1.region1.zone2"
 ```
 
-If no servers are available, the request may return with a failure.
+Use an asterisk (*) to specify all zones in a region. (You can't do this for region or cloud.) For example:
+
+```go
+"postgres://username:password@localhost:5433/database_name?load_balance=true& \
+    topology_keys=cloud1.region1.*"
+```
 
-You still need to specify load balance as true to enable the topology-aware connection load balancing.
+#### Fallback topology keys
+
+To specify fallback locations in cases where a location is unavailable, add `:n` to the topology key, where n is an integer indicating priority. The following example sets `zone1` as the topology key, and zones 2 and 3 as fallbacks (in that order) if `zone1` can't be reached:
```

Comment:
IMO, we can introduce the fallback use case before the syntax. Something like:
So users can specify fallback placements when they specify topology-keys. So when no nodes in the primary placements are available, the driver can attempt connections to nodes in the fallback placements.
Also, if topology-keys without preference-value/priority is provided (i.e. no explicit fallback placement is specified), the driver falls back to nodes in the entire cluster in case the nodes in those specified placements are not available.